pkg/controller/acmeorders/sync.go

```diff
@@ -188,12 +188,17 @@ func (c *controller) createOrder(ctx context.Context, cl acmecl.Interface, o *cm
 	}
```

Comment:
I see there are some existing sync_test.go tests. Maybe add some unit tests for this new feature.